helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -49,65 +49,44 @@ data:
 
         [http.routers.gateway]
           entryPoints = ["http"]
-          Rule = "PathPrefix(`{{ .Values.gatewayServicePrefix | default "/api/" }}auth`)"
```

Comment:
Default has been moved out of the templates to values.yaml, `clean` is used to clean paths, eg `/some//things/ -> /some/things` or ``// -> `.